src/sentry/south_migrations/0388_auto__add_index_file_checksum.py

```diff
@@ -0,0 +1,1055 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```

Comment:
Because creating this index will take a while, especially with `CONCURRENTLY`, this needs to be flipped to `True` so we don't block deploy for however long it takes to create the index.